common/src/sql/dbinit.sql

```diff
@@ -442,11 +442,8 @@ CREATE TABLE omicron.public.image (
     project_id UUID NOT NULL,
     volume_id UUID NOT NULL,
 
-    /* Optional URL: Images may be backed by either a URL or a volume */
     url STRING(8192),
```

Comment:
I see you removed the "optional" comments here but the fields _are_ still optional.  Is that the intent or are they supposed to be `NOT NULL`?

Reply:
They are supposed to be optional